FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/portable/NetworkInterface/esp32/NetworkInterface.c

```diff
@@ -44,6 +44,10 @@ enum if_state_t {
 static const char *TAG = "NetInterface";
 volatile static uint32_t xInterfaceState = INTERFACE_DOWN;
 
+#if ( ipconfigHAS_PRINTF != 0 )
```

Comment:
What else switches on when you have this defined? It seems to me like we need to be able to switch off this monitoring printouts and keep the Printf definition for other purposes?

Reply:
Would be better to make prvMonitorResources() a macro that only exists when ipconfigHAS_PRINTF is defined as != 0 rather than guard each call to prvMonitorResources() with its own #if block.

Reply:
Yes I agree, less opportunity to miss something.

Reply:
Agreed.
But this (copying of one file at a time) is not a good was to achieve SSoT. Let this PR stay open until something is decided. Then if necessary, I will make the changes